test/PooltogetherGovernor.t.sol

```diff
@@ -3,12 +3,19 @@ pragma solidity ^0.8.18;
 
 import {IERC20} from "@openzeppelin/contracts/interfaces/IERC20.sol";
 import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {ERC20VotesComp} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20VotesComp.sol";
```

Comment:
*arbitrary comment location*

Looks like this needs a run of the linter, some things in the test file to fix.

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/15/commits/0c652d47811e76412b32af2e1a38ab52b2566ef5

Reply:
New ones showing up now 🙈

Reply:
Should be good now